sechub-web-ui-solution/helm/sechub-web-ui/templates/deployment.yaml

```
@@ -0,0 +1,90 @@
+# SPDX-License-Identifier: MIT
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 49%
+  selector:
+    matchLabels:
+      name: {{ .Values.name }}
+  template:
+    metadata:
+      labels:
+        name: {{ .Values.name }}
+{{- if .Values.templateMetadataAnnotations }}
+      annotations:
+        {{ .Values.templateMetadataAnnotations | indent 8 | trim }}
+{{- end }}
+    spec:
+      securityContext:
+        runAsUser: 4242  # uid of the application user. (should be same as in the Dockerfile)
+        runAsGroup: 4242 # gid of the application group. (should be same as in the Dockerfile)
+        fsGroup: 4242    # This gid has write access to the mounted volumes.
+{{- if .Values.go_mmproxy.enabled }}
+      initContainers:
+        - name: setup
+          image: "{{ .Values.go_mmproxy.image }}"
+          command:
+            - "/bin/sh"
+            - "-cx"
+          args:
+            - |
+              /sbin/ip rule add from 127.0.0.1/8 iif lo table 123
+              /sbin/ip route add local 0.0.0.0/0 dev lo table 123
+          securityContext:
+            capabilities:
+              add:
+                - NET_ADMIN
+{{- end }}
+      containers:
+{{- if .Values.go_mmproxy.enabled }}
+      # go-mmproxy sidecar container
+      - name: go-mmproxy
+        image: "{{ .Values.go_mmproxy.image }}"
```

Please see the comment in `values.yaml`
```suggestion
        image: "{{ .Values.go_mmproxy.image }}"
        imagePullPolicy: {{ .Values.imagePullPolicy }}
```